src/core/best-practices.js

```diff
@@ -21,7 +21,9 @@ export function run() {
       bp.textContent
     }</li>`;
     ul.appendChild(li);
-    bp.prepend(document.createTextNode(`Best Practice ${num}: `));
+    ul.classList.add("practicebox");
+    const title = hyperHTML`<span><span class="practicetitle">Best Practice ${num}</span>: </span>`;
```

Comment:
Nested spans are probably not needed here... show you a trick 🎩 

```suggestion
    const title = hyperHTML`<span class="practicetitle">Best Practice ${num}</span>: `;
```